src/themes/website/styles/forms.scss

```diff
@@ -281,4 +281,14 @@ $search-button-width: 30px;
 }
 .input-password::-ms-reveal {
     display: none;
-}
\ No newline at end of file
+}
+
+.remove-button {
+    display: flex;
+    align-items: center;
+    height: 100%;
+}
+
+.remove-inline {
+    padding-bottom: 1rem;
```

Comment:
`padding-bottom: 1rem;` - are you sure about using `rem` here? This basically means that your padding will use the root element font size (some [useful reading](https://medium.com/@madhum86/css-font-sizing-pixels-vs-em-vs-rem-vs-percent-vs-viewport-units-b1485716afe7)).